tailwind.config.js

```diff
@@ -8,7 +8,12 @@ module.exports = {
   content: [
     './wagtail/**/*.{py,html,ts,tsx}',
     './wagtail/**/static_src/**/*.js',
-    './client/**/*.{js,ts,tsx,mdx}',
+    // Make sure NOT to include the `client/scss` directory,
+    // even if we don't specify `*.scss` files here.
+    // The directory would still be scanned for files, which would cause
+    // the styles to rebuild in a loop.
+    // https://tailwindcss.com/docs/content-configuration#styles-rebuild-in-an-infinite-loop
+    './client/src/**/*.{js,ts,tsx,mdx}',
```

Comment:
There are other directories in `client` other than `src` and `scss`, which are `storybook` and `tests`. I don't think we need to watch those, but if it's desired we can use either of the following instead:

- `'./client/[!scss]/**/*.{js,ts,tsx,mdx}'`
- `'./client/{src,storybook,tests}/**/*.{js,ts,tsx,mdx}'`

That still doesn't include the files within `client` itself I think, but I don't think they're important...